Lesson25/Fırat/script.js

```diff
@@ -5,12 +5,37 @@ h2 (.post-title)
 p (.post-body)
 */
 
-const URL = 'https://jsonplaceholder.typicode.com/posts';
+const URL = "https://jsonplaceholder.typicode.com/posts";
+const postList = document.getElementById("post-list");
 function fetchPosts() {
+  fetch(URL)
+    .then((response) => response.json())
+    .then((posts) =>
+      posts.forEach((post) => {
+        const listItem = document.createElement("li");
+        listItem.classList.add("post");
+
+        const postTitle = document.createElement("h2");
+        postTitle.classList.add("post-title");
+        postTitle.innerText = post.title;
+        listItem.appendChild(postTitle);
+
+        const postBody = document.createElement("p");
+        postBody.classList.add("post-body");
+        postBody.innerText = post.body;
+        listItem.appendChild(postBody);
+
+        postList.appendChild(listItem);
+      })
+    )
+    .catch((error) => {
+      console.error("hata:", error);
+    });
 }
 
 function clearPosts() {
-};
+  postList.innerHTML = " ";
```

Comment:
Please use innerText instead of innerHTML, see https://medium.com/@verity.carlos/why-you-shouldnt-use-innerhtml-and-what-to-use-instead-ed99d064a416